src/args.ts

```diff
@@ -19,20 +20,35 @@ function determineToolchain(overrideFile: string): string {
         return toolchainInput;
     }
 
-    if (!existsSync(overrideFile)) {
+    const toolchainPath = existsSync(overrideFile)
+        ? overrideFile
+        : existsSync(`${overrideFile}.toml`)
+        ? `${overrideFile}.toml`
+        : undefined;
+
+    if (!toolchainPath) {
         throw new Error(
             "toolchain input was not given and repository does not have a rust-toolchain file"
         );
     }
```

Comment:
Need warn it both files existing.

Reply:
Sorry for the delay on this, I'll address it today.